src/components/SDCWrapper.tsx

```diff
@@ -11,7 +11,9 @@ const SDCWrapper: React.FC<SDCWrapperProps> = (props): JSX.Element => {
     return (
         <Box minHeight={"100vh"} position={"relative"}>
             <Navbar session={props.session} />
-            <Box pb={DEFAULT_FOOTER_HEIGHT}>{props.children}</Box>
+            <Box pb={DEFAULT_FOOTER_HEIGHT} px="48">
```

Comment:
why is the `px=48` change needed? if you needed it specifically for the login/verify pages, could you add it there instead? changes here should be made if they're supposed to apply to everything on the website since basically everything is gonna be wrapped in this

Reply:
Actually I think this makes sense, but we need to be careful when merging this because some of the other components have `px=48` to align them with the footer/navbar, so once this gets merged I think it's gonna double the padding

Reply:
Are those PRs already in staging?